src/lib/nvme_pt_ioctl.c

```diff
@@ -727,6 +733,94 @@ static int validate_sessn(struct opal_device *dev)
 	return 0;
 }
 
+static int  parse_tper_host_prop(struct sed_device *device)
```

Comment:
please remove extra space between return type and function name

Reply:
will do it